lasagne/layers/conv.py

```diff
@@ -117,23 +109,22 @@ class Conv1DLayer(Layer):
         An integer or a 1-element tuple specifying the stride of the
         convolution operation.
 
-    border_mode : str, one of 'valid', 'full', 'same'
-        A string indicating the convolution border mode.
+    pad : {0, (0,), 'full', 'same'} (default: 0)
```

Comment:
Ah, ok. I think `pad=None` would be a little clearer, but `pad=0` works also. I do think it's confusing though to document multiple ways to to specify the same thing (`pad=0`, `pad=(0,)` and `pad=(0,0)` for the Conv2DLayer), even if they are all accepted.